option.go

```diff
@@ -101,3 +101,10 @@ func WithAttributesGetter(attributesGetter AttributesGetter) Option {
 		cfg.AttributesGetter = attributesGetter
 	})
 }
+
+// WithInstrumentAttributesGetter takes InstrumentAttributesGetter that will be called every time metric is recorded.
```

Comment:
```suggestion
// WithInstrumentAttributesGetter takes InstrumentAttributesGetter that will be called every time metric is recorded to instruments.
```

Same as mentioned above.